### PR TITLE
update copyright notices to 2020

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
Because copyright notices usually include the year, and that has just changed, this PR attempts to update all DA copyright notices from 2019 to 2020 in this repo.

### CODE REVIEW

**Comment by bame-da** [comment]
Should it be just `2020` or `201X - 2020`? Maybe worth checking whether just including `2020 ` has any legal ramifications.

**Comment by garyverhaegen-da** [comment]
Manoj says 2020 is fine; headers need to be updated on first change ideally, so this should be the first PR we merge this year.

File: daml-lf/archive/src/main/protobuf/com/digitalasset/daml_lf_1_6/daml_lf.proto [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// Copyright (c) 2020 The DAML Authors. All rights reserved.
```
Comment (cocreature) [inline daml-lf/archive/src/main/protobuf/com/digitalasset/daml_lf_1_6/daml_lf.proto]: Given that the non-dev DAML-LF proto files are supposed to be frozen, I’m not sure it makes sense to update the copyright header. I thought we even have a test that the hash of these files doesn’t change.

Comment (garyverhaegen-da) [reply daml-lf/archive/src/main/protobuf/com/digitalasset/daml_lf_1_6/daml_lf.proto]: Indeed. Reverting that. I assume that's only for numbered versions, i.e. not the dev ones?

Comment (cocreature) [reply daml-lf/archive/src/main/protobuf/com/digitalasset/daml_lf_1_6/daml_lf.proto]: Yeah the dev proto files change.

File: README.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ This repository hosts all code for the [DAML smart contract language and SDK](ht
 To download DAML, follow [the installation instructions](https://docs.daml.com/getting-started/installation.html).
 Once installed, to try it out, follow the [quickstart guide](https://docs.daml.com/getting-started/quickstart.html).
 
-If you have questions about how to use DAML or how to build DAML-based solutions, please ask them 
+If you have questions about how to use DAML or how to build DAML-based solutions, please ask them
```
Comment (dasormeter) [inline README.md]: whitespace change

File: README.md [hunk 3]
```diff
@@ -71,7 +71,7 @@ We have a single script to build most targets and run the tests. On Linux and Ma
 
 To just build do `bazel build //...`, and to just test do `bazel test //...`. To read more about Bazel and how to use it, see [the Bazel site](https://bazel.build).
 
-On Mac if building is causing trouble complaining about missing nix packages, you can try first running `nix-build -A tools -A cached nix` repeatedly until it completes without error. 
+On Mac if building is causing trouble complaining about missing nix packages, you can try first running `nix-build -A tools -A cached nix` repeatedly until it completes without error.
```
Comment (dasormeter) [inline README.md]: whitespace change

**Comment by garyverhaegen-da** [comment]
> script looks to have made a bunch of whitespace changes to NOTICES file which i don't think are needed

Reverted. Please don't list them all.

File: README.md (hunk 2 above)
Comment (garyverhaegen-da) [reply README.md]: Reverted.

File: README.md (hunk 3 above)
Comment (garyverhaegen-da) [reply README.md]: Reverted.

**Comment by hurryabit** [comment]
Thanks for taking care of this.

**Comment by stefanobaghino-da** [comment]
And happy new year! :tada:

**Comment by garyverhaegen-da** [comment]
Note: adding Standard-Change label as this technically changes the infra files, though only in their comments. I'll also look into why this was not caught by CI.

**Comment by garyverhaegen-da** [comment]
Also recording that I have applied those changes today (2020-01-22).

**Pull request merged**